### DISCUSSION TITLE
[M-Z] (2024/25) - sez. 0.1 - es. 1.12 (30 settembre)

### DISCUSSION THREAD

**Discussion by marcoseverini** [sapienzastudentsnetwork/calcolo-delle-probabilita#281] [Esercizi M-Z]
<img width="696" alt="immagine" src="https://github.com/user-attachments/assets/b025d9bf-e88d-412e-a917-c2805f366d13">

**Comment by marcoseverini** [comment]
<img width="725" alt="immagine" src="https://github.com/user-attachments/assets/a300a767-2afe-4f1e-a4e2-285244171a1d">

**Comment by AglaiaNorza** [comment]
a) $30^5$ (30 possibilità per il primo premio, 30 per il secondo...)
b) 30 x 29 x 28 x 27 x 26
